paper-typeahead-input.html

```diff
@@ -473,8 +504,8 @@
       }
     },
     _search:function(term){
-      if (term == ""){
-        this._suggestions = [];
+      if (term == "" && !this.isAllowSuggestOnEmptyInput){
```

Comment:
Okay. This makes sense.  And it's good to have it default to be false. 
Can you rename it to be suggestOnEmptyInput instead of isAllowSuggestOnEmptyInput. So that to be consistent with the other property suggestOnFocus.